internal/storetest/localstore.go

```diff
@@ -72,30 +72,20 @@ func initLocalStore(
 
 func getLocalServerModelAndTuples(
 	storeData *StoreData,
-	basePath string,
+	format authorizationmodel.ModelFormat,
 ) (*server.Server, *authorizationmodel.AuthzModel, error) {
 	var fgaServer *server.Server
 
 	var authModel *authorizationmodel.AuthzModel
 
-	format, err := storeData.LoadModel(basePath)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	err = storeData.LoadTuples(basePath)
```

Comment:
Loading the tuples was added in #212 so we'd need to ensure that is maintained.

Reply:
Nice catch, missed it resolving merge conflicts.